.github/workflows/release.yaml

```diff
@@ -47,6 +47,50 @@ jobs:
 
           script/release
 
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: ./dist/
+
+  upload-release-assets:
+    name: Upload release assets
+    runs-on: ubuntu-latest
+    needs: release
+    if: github.event_name == 'release'
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Download build artifacts
+        uses: actions/download-artifact@v2
```

Comment:
```suggestion
```
This workflow only runs on release.

Reply:
True, however the current setup would make it more universal. We could continue using it even if, for example, nightly builds on a cron schedule are added. It doesn't add much overhead either. Lastly, the job config could be used elsewhere much more easily this way, say for the Core repo.